src/services/bids/models/BidGetItem.php

```diff
@@ -64,5 +64,5 @@ class BidGetItem extends Model
     /**
      * @var AuctionBidItem
```

Comment:
I think it should be this way:
```suggestion
     * @var AuctionBidItem[]
```